src/lib/flags.js

```
@@ -2,7 +2,7 @@ import flag from 'cozy-flags'
 
 export const initFlags = () => {
   let activateFlags = flag('switcher') === true ? true : false
-
+  console.log('activateFlags', activateFlags)
```

ESLint

[error] Unexpected console statement.

(no-console)